spec/dummy/db/schema.rb

```diff
@@ -173,6 +173,18 @@
     t.string   "base_url",                                     null: false
   end
 
+  create_table "atmosphere_migration_jobs", force: true do |t|
+    t.integer  "appliance_type_id"
+    t.integer  "compute_site_source_id"
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.